VideoLocker/res/layout/row_course_dashboard_list.xml

```diff
@@ -1,36 +1,36 @@
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    android:id="@+id/chapter_row_layout"
+    xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
-    android:id="@+id/chapter_row_layout"
     android:layout_width="match_parent"
-    android:layout_height="@dimen/dashboard_navigation_element_height"
+    android:layout_height="@dimen/dashboard_nav_element_height"
```

Comment:
Ideally, we should hardcode the `minHeight` and set the `layout_height` to `wrap_content`, with some reasonable vertical padding (maybe the same as the horizontal padding if that fits within the existing height), so that it can scale to a larger size if the text wraps in devices with small screens.